client/my-sites/site-settings/disconnect-site/missing-feature.jsx

```diff
@@ -60,7 +60,10 @@ class MissingFeature extends PureComponent {
 					<Button
 						disabled={ isEmpty( this.state.tokens ) }
 						href={ addQueryArgs(
-							{ 'missing-features': this.state.tokens.map( this.normalizeToken ) },
+							{
+								reason: 'missing-feature',
+								text: this.state.tokens.map( this.normalizeToken ),
```

Comment:
Maybe some value in sorting the tokens here so identical sets of reasons always match up in the data?